Silicon/ElkhartlakePkg/FspBin/FspBin.inf

```diff
@@ -0,0 +1,26 @@
+## @file
+#  File to describe FSP repo information
+#
+#  Copyright (c) 2020 - 2021, Intel Corporation. All rights reserved.<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#
+##
+
+
+[UserExtensions.SBL."CloneRepo"]
+  REPO    = https://github.com/intel/FSP.git
+  COMMIT  = 7954a8db808942a7430a00f4e5a757eb84951edc
+
+[UserExtensions.SBL."CopyList"]
+  ElkhartLakeFspBinPkg/FspBin/FSPRel.bin               : Silicon/ElkhartLakePkg/FspBin/FspDbg.bin
+  ElkhartLakeFspBinPkg/FspBin/FSPRel.bin               : Silicon/ElkhartLakePkg/FspBin/FspRel.bin
+  ElkhartLakeFspBinPkg/FspBin/FSP.bsf                  : Silicon/TigerlakePkg/FspBin/Fsp.bsf
+  ElkhartLakeFspBinPkg/Include/FspUpd.h                : Silicon/ElkhartLakePkg/Include/FspUpd.h
+  ElkhartLakeFspBinPkg/Include/FsptUpd.h               : Silicon/ElkhartLakePkg/Include/FsptUpd.h
+  ElkhartLakeFspBinPkg/Include/FspmUpd.h               : Silicon/ElkhartLakePkg/Include/FspmUpd.h
+  ElkhartLakeFspBinPkg/Include/FspsUpd.h               : Silicon/ElkhartLakePkg/Include/FspsUpd.h
+  ElkhartLakeFspBinPkg/Include/MemInfoHob.h            : Silicon/ElkhartLakePkg/Include/MemInfoHob.h
+  ElkhartLakeFspBinPkg/VbtBin/Vbt.bin                  : Platform/ElkhartLakeBoardPkg/VbtBin/Vbt.dat
+  ElkhartLakeFspBinPkg/VbtBin/Vbt.bsf                  : Platform/ElkhartLakeBoardPkg/VbtBin/Vbt.bsf

```

Comment:
The build issue should be caused by the folder name.
"ElkhartLakePkg" should be "ElkhartlakePkg" in Silicon folder.

Reply:
Updated, thanks for the review.